root/tmva/tmva/ConvNetBenchmarks.h

```diff
@@ -0,0 +1,105 @@
+#include "TMVA/Factory.h"
+#include "TMVA/DataLoader.h"
+#include "TMVA/Config.h"
+#include "TMVA/DataSetInfo.h"
+#include "TFile.h"
+#include "TTree.h"
+
+
+void CNN_ECAL_test(TString archName) {
+
+    int ntrainEvts = 5000;
+    int ntestEvts =  5000;
+
+    ROOT::EnableImplicitMT(0);
+    TMVA::Config::Instance();
+
+    TFile *input(0);
+
+    TString fname = "/media/manos/D/Work/rootbench/root/tmva/tmva/EcalData_n50000.root";
```

Comment:
Do we need to read an external file at all (maybe, but  I want to pose the question). Another option would be to generate a root tree on the fly. This would mean less dependencies, so easier maintenance in the long run.

Reply:
This is also what @lmoneta proposed and I agree, it seems like a great idea. Benchmarks don't need to be useful after all.

Reply:
@steremma I will be strongly in favour to have this file generated on the fly :)